pulseapi/profiles/migrations/0022_userprofile_created_at.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-06-28 00:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.utils.dateparse import parse_date
+from django.utils.timezone import make_aware
+
+def default_start_time():
+    date_str = "2016-01-26"
+    default_date = parse_date(date_str)
+    return make_aware(default_date)
```

Comment:
I think `default_date` needs to be a `datetime` object not a `date` object...that's probably why travis is failing as well.

You probably want to use something like `parse_datetime('2016-01-26 00:00:00')` as the default date instead and then you can make it timezone aware.

Reply:
@Pomax Can you get this over the edge?

Reply:
yar